microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -23,63 +23,55 @@ ms.date: 02/24/2023
-
-
-<!-- ------------------------------ -->
-#### Firefox Responsive Design View
-
-Firefox has a [responsive design view](https://developer.mozilla.org/docs/Tools/Responsive_Design_View) that encourages you to stop thinking in terms of specific devices and instead explore how your design changes at common screen sizes, or on your own screen size by dragging the edges of the window.
+Browser emulators are great for testing the way your site looks and behaves at different screen sizes and resolutions, or when touch events are enabled, or when the user-agent string is set to another browser's user-agent string.
 
+You use browser emulators from the browser you're developing on. For example, in Microsoft Edge, you can use the **Device Emulation** tool in DevTools to get an approximation of how your site looks and behaves on different devices.  See [Emulate mobile devices (Device Emulation)](./index.md) for more information on how to use the **Device Emulation** tool in Microsoft Edge DevTools.
 
-<!-- ------------------------------ -->
-#### EdgeHTML emulation
+Browser emulators don't emulate differences in web API or CSS support. For example, the **Device Emulation** tool in Microsoft Edge doesn't emulate how the different web APIs or CSS features that your site might be using are supported in Firefox or Safari. Test your site on browsers running on real devices, to be certain everything behaves as expected.
 
-To emulate Windows Phones, use the Microsoft Edge (EdgeHTML) [built-in emulation](/archive/microsoft-edge/legacy/developer/devtools-guide/emulation).
+Here are some browser emulators you can use to test your site on other browsers:
 
-Use [IE 11 Emulation](/previous-versions/windows/internet-explorer/ie-developer/samples/dn255001(v=vs.85)) to simulate how your page might look in older versions of Internet Explorer.
+* [Firefox Responsive Design Mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mode/index.html).
+* [Chrome Device Mode](https://developer.chrome.com/docs/devtools/device-mode).
+* [Safari Responsive Design Mode](https://developer.apple.com/documentation/safari-developer-tools/responsive-design-mode).
 
 
 <!-- ====================================================================== -->
 ## Device emulators and simulators
 
-Device simulators and emulators simulate not just the browser environment but the entire device.  Each simulator is useful to test things that require OS integration, such as form input with virtual keyboards.
+Device simulators and emulators simulate not just the browser environment but the entire device.  Each simulator is useful to test things that require specific integration with the operating system, such as form input with mobile device keyboards.
```

Comment:
```suggestion
Device simulators and emulators simulate the entire device, rather than just simulating the browser environment.  Each simulator is useful to test things that require specific integration with the operating system, such as form input with mobile device keyboards.
```
avoiding negative lead-in 
@captainbrosset

Reply:
applied locally, will commit/push